src/content_sao.cpp

```diff
@@ -204,6 +204,16 @@ void UnitSAO::getAttachment(int *parent_id, std::string *bone, v3f *position,
 	*rotation = m_attachment_rotation;
 }
 
+void UnitSAO::detach()
```

Comment:
That make sense, but can you rename it to detachFromParent instead of too generic name?